hw/top_earlgrey/ip/sensor_ctrl/data/sensor_ctrl.prj.hjson

```diff
@@ -11,7 +11,7 @@
       {
         version:            "1.0",
         life_stage:         "L1",
-        design_stage:       "D2",
+        design_stage:       "D3",
```

Comment:
sorry i can't remember what we said before..since we're at D3 now should we include a commit hash? Especially since this block doesn't have any V* stages.

Reply:
This would be optional at this point, I believe.
We need to record the commit hash when regressing any of the dev stages (combined with increasing the version number), or when we officially sign off the IP to the L2 stage.

From https://docs.opentitan.org/doc/project/development_stages/#versioning:
```
Once a design has completed all stages for a product feature set, 
the sign-off process intends to end all development for that design. 
Its Life Stage should transition to sign-off after the review process, 
and no more modifications should be made. The commit ID of the 
version that was signed off is recorded in the commit_id field of the 
.prj.hjson file.
```

Reply:
To that end, I actually wonder whether we want to programmatically derive the commit hash in the future instead of hardcoding it in this file. I mean, technically it should be possible to retrieve the commit hash automatically via git blame and some scripting around it that checks when the last update has been made to the `*_stage` keys in this file.

Reply:
o yeah good point. That would be nice, so we just advance the "L*" and let everything take care of itself. 
But specifically for this block..since there's no block level and it's tied only to top, i guess we should wait until it's "tapeout ready"?

Reply:
Yeah - I think it would make sense to wait until L2 for this.

I believe the automatic commit ID extraction could be useful, since it removes the ambiguity about whether this field should be filled in or not. And it would also solve some consistency issues with the hand-maintained commit IDs (I have seen cases where the field is not up to date, or where the commit ID does not exist in the tree).